item_view.h

```diff
@@ -81,9 +81,9 @@ class ItemView : public Gtk::Grid {
 
         if (conceal && !isTOTP)
             value_widget->set_visibility(false);
-        attach(*value_widget, 1, my_index, conceal ? 1 : 3, 1);
+        attach(*value_widget, 1, my_index, conceal || copy_button ? 1 : 3, 1);
 
-        if (conceal) {
+        if (copy_button) {
```

Comment:
Again, since this is just always true, I don't think it needs to be in a conditional